### PR TITLE
Feat: home page

### PR DESCRIPTION
**Pull request by bakhosRachwan**
almost done still missing the what we do component and the header

### CODE REVIEW

**Comment by osamaakb** [comment]
@bakhosRachwan please add them ASAP so we can merge this

File: src/Pages/HomePage/Homepage.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import NeedATourGuide from '../../Components/NeedATourGuide/NeedATourGuide';
+import CityCard from '../../Components/Card/CityCard/CityCard';
+import JoinOurTeam from '../../Components/Card/JoinOurTeam/JoinOurTeam.js';
+import ActivityCard from '../../Components/Card/ActivityCard/ActivityCard';
+import { Suspense } from 'react';
+import { Carousel } from 'antd';
+import { LeftCircleOutlined, RightCircleOutlined } from '@ant-design/icons';
+
+const HomePage = () => {
+  const guides = [
```
Comment (osamaakb) [inline src/Pages/HomePage/Homepage.js:15]: can you move this to a folder in the `src` name it `mockdata` and inside it add file called `data.js` and then export this array from there so we can keep it clean here

**Review by osamaakb — CHANGES_REQUESTED** [review]

**Comment by osamaakb** [comment]
@bakhosRachwan I merged the header, I think now you have everything to finish this page?

**Comment by bakhosRachwan** [comment]
can we close and delete this branch and just merge the other one i pushed

**Pull request closed without merging**